foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -15,6 +15,17 @@
           }
         ]
       },
+      {
+        "name": "it-infrastructure-java-upgrade-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_10_0.ItInfrastructureJavaUpgradeMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "*-tests/*-tests-docker/src/main/resources/docker/Dockerfile"
```

Comment:
A: In the past an overly-specific path has caused more headache than help. Might make sense to do `*-tests/**/Dockerfile`